util/test/test_dftool.py

```diff
@@ -118,26 +118,8 @@ def test_manyrandom():
 def test_noarg_commands():
     # Test commands that have no args. They're usually help commands;
     # sometimes they do the main work (eg compile).
-    argv1s = [
-        "",
-        "query",
-        "volsym",
-        "getrate",
-        "calc",
-        "dispense",
-        "dispense_active",
-        "querymany",
-        "compile",
-        "manyrandom",
-        "newdfrewards",
-        "mine",
-        "newacct",
-        "newtoken",
-        "acctinfo",
-        "chaininfo",
-    ]
```

Comment:
This should not be removed. It's really important to keep, since it gives at least super-basic tests to many dftool CLI tools that don't otherwise tests.

Instead, just simply remove the lines for "query" and "dispense"

Reply:
More info:

In slack @idiom-bytes had mentioned that they were removed "because they were not doing anything". This is incorrect. Ask yourself "why might these possibly be here?" I'll make life simple and give the answer: `dftool X` with no extra arguments goes to that method, and gives the _help_ info for that command. And that's what this is testing. 

Extra benefit: many dftool commands _only_ have this as a test, so this is the only pushes down into that dftool command's `do_X()` function.

Reply:
Yup, as discussed in our meeting. Thank you.